packages/insured-bridge-relayer/src/CrossDomainFinalizer.ts

```diff
@@ -41,7 +41,7 @@ export class CrossDomainFinalizer {
     readonly crossDomainFinalizationThreshold: number = 5
   ) {}
   async checkForBridgeableL2TokensAndBridge() {
```

Comment:
modifications in this file was made to make the logs produced by this utility fit on one line.